### DISCUSSION TITLE
Aiman

### DISCUSSION THREAD

**Discussion by ALatif786** [community/community#134548] [Other Feature Feedback, Questions, & Ideas]
Guide me through the process of creating my own AI based speech generator from the scratch

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @ALatif786!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Models category. I've gone ahead and moved it for you. Good luck!
